pkg/controller/component/runtimes.go

```diff
@@ -16,13 +16,13 @@ const (
 // todo: extract into configuration file
 var registry = RuntimeRegistry{
 	runtimes: map[string]runtimeVersions{
-		"spring-boot":     newRuntime("quay.io/halkyonio/hal-maven-jdk"),
-		"vert.x":          newRuntime("quay.io/halkyonio/hal-maven-jdk"),
-		"quarkus":         newRuntime("quay.io/halkyonio/hal-maven-jdk"),
-		"thorntail":       newRuntime("quay.io/halkyonio/hal-maven-jdk"),
-		"openjdk8":        newRuntime("registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift"),
-		"node.js":         newRuntime("nodeshift/centos7-s2i-nodejs"),
-		supervisorImageId: newRuntime("quay.io/halkyonio/supervisord"),
+		"spring-boot":     newRuntime("quay.io/halkyonio/hal-maven-jdk", "*"),
+		"vert.x":          newRuntime("quay.io/halkyonio/hal-maven-jdk", "*-all"),
+		"quarkus":         newRuntime("quay.io/halkyonio/hal-maven-jdk", "*-runner"),
+		"thorntail":       newRuntime("quay.io/halkyonio/hal-maven-jdk", "*"),
```

Comment:
I think this could/should be `*-thorntail`.

Reply:
Indeed, though it doesn't really matter since the build only generates one jar… :)

Reply:
It only generates one JAR if `<packaging>war</packaging>`. That is 99% of cases, probably, but people can have `<packaging>jar</packaging>`.

Reply:
Noted and fixed! Thank you.